blockchain/v2/routine.go

```diff
@@ -98,7 +98,9 @@ func (rt *Routine) start() {
 			return
 		}
 		rt.metrics.EventsOut.With("routine", rt.name).Add(1)
-		rt.logger.Debug(fmt.Sprintf("%s: produced %T %+v", rt.name, oEvent, oEvent))
+		if rt.logger.IsDebugging() {
```

Comment:
These are fine, having said that I believe that the blockchain/v2 reactor is never used.

Reply:
Eh.. I like having stuff consistently applied even if just hanging out waiting for `/dev/null`